pootle/apps/contact/forms.py

```diff
@@ -18,10 +18,10 @@
 
 class ContactForm(MathCaptchaForm, OriginalContactForm):
 
-    field_order = ['name', 'email', 'subject', 'body', 'captcha_answer',
+    field_order = ['name', 'email', 'summary', 'body', 'captcha_answer',
```

Comment:
Names are chosen to prevent confusion, I only reported this cause I actually was confused and checked.  I expect the next person to review to be similarly confused.  You changed it because of other code conflict, not because its not quite a subject.